src/WellIO.jl

```diff
@@ -1,5 +1,26 @@
 module WellIO
 
-# Write your package code here.
+using FileIO
+using Tables
 
+
+function load(fname::File{format"LAS"})
+    s = open(fname) do file
+        read(file,String)
+    end
+end
+
+struct Well
+    VersionNumber::Float
+    wrap::Bool
+    wellInfo::Dict()
+    curveInfo::Dict()
+    otherInfo::String
+    asciiData::Tables
```

Comment:
Camel case is not common in Julia. Can you remove the upper case letters in the middle of the identifers?

Also, I think you need to review basic Julia because `Dict()` is not a type specification, it is an instance of a dictionary. Have you tried loading your package at least before asking for a review?